README.md

```diff
@@ -233,6 +233,8 @@ export LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=`git config github.user`
 export LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=`git config github.token`
 export LAUNCHER_MISSIONCONTROL_GITLAB_USERNAME=`git config gitlab.user`
 export LAUNCHER_MISSIONCONTROL_GITLAB_PRIVATE_TOKEN=`git config gitlab.token`
+export LAUNCHER_MISSIONCONTROL_BITBUCKET_USERNAME=BitbucketFakeUserName
+export LAUNCHER_MISSIONCONTROL_BITBUCKET_APPLICATION_PASSWORD=BitbucketFakeApplicationPassword1234$
```

Comment:
Why this? Why not the same as the others?

Reply:
It's not a token, it's a Application Password as they call it. It's used with basic auth, not as a Bearer token.